### ISSUE TITLE
[FEAT] better support for callout

### ISSUE THREAD

**Issue by Rsssstt** [st3v3nmw/obsidian-spaced-repetition#583]
**Is your feature request related to a problem? Please describe.**
Yes, when I use spaced repetition with a callout, I get in trouble. If I use single-line syntax in a callout (in a second level list), latex will not render correctly. Then I think maybe I can let the whole callout to be a card with a multi-line syntax, but I still fail.

**Describe the solution you'd like**
I think make a whole callout to be a flashcard will be a good idea.

**Additional context**
Some other plugin's function will not render correctly, like obsidian-tikzjax. Yes I'm using spaced repetition to study my mathematics note 😁! Thanks for the efforts of the developers.

**Comment by monstertank** [comment]
try this snippet
`#sr-easy-btn {
    background-color: #344835a8;
}


#sr-hard-btn, #sr-good-btn, #sr-easy-btn {
    height: 48px;
    margin: auto;
    color: #8f8d8d;
    
}
#sr-show-answer {
    height: 48px;
    line-height: 48px;
    width: 96%;
    text-align: center;
    position: absolute;
    bottom: 0;
    cursor: pointer;
    background-color: #23282f;
    color: #aeafb1;
    border-radius: 4px;
    user-select: text;
}
#sr-good-btn {
    background-color: #2b3a46;
}

#sr-hard-btn {
    background-color: #402e2d;
 
}

.modal-content.sr-modal-content {
    position: unset !important;
}
.sr-link {
    float: unset !important;
}
#sr-flashcard-view {
  
    padding-right: 10px;
}`

**Comment by Rsssstt** [comment]
Thanks for your help!

**Comment by ronzulu** [comment]
Hi @Rsssstt 

I'm wondering if the help provided by @monstertank solves the problem?

If not, please attach a markdown file that demonstrates the problem, as well as an annotated screenshot of what currently happens and what you prefer.

Cheers
Ronny
